lib/site-inspector.rb

```diff
@@ -1,24 +1,19 @@
-require 'nokogiri'
+
+# needed for HTTP analysis
 require 'open-uri'
-require 'public_suffix'
-require 'gman'
-require 'net/http'
-require "dnsruby"
-require 'yaml'
-require 'sniffles'
 require "addressable/uri"
+require 'public_suffix'
 require 'typhoeus'
-require 'json'
-require 'resolv'
 
 require_relative 'site-inspector/cache'
+require_relative 'site-inspector/headers'
 require_relative 'site-inspector/sniffer'
 require_relative 'site-inspector/dns'
 require_relative 'site-inspector/compliance'
-require_relative 'site-inspector/headers'
+
 
 if ENV['CACHE']
-  Typhoeus::Config.cache = SiteInspectorDiskCache.new(ENV['CACHE'])
+  Typhoeus::Config.cache = SiteInspectorDiskCache.new(ENV['CACHE'], ENV['CACHE_REPLACE'])
```

Comment:
Should likely call `Dotenv.load` to read `.env` files here if we're using Env vars.